plugin.json

```diff
@@ -1,10 +1,10 @@
 {
     "id": "tesseract-ocr",
```

Comment:
We keep the same id so that the new version still works with old recipes, right ?

Reply:
yes and changing the plugin id will create a duplicated plugin because the plugin-id is the name of the plugin folder